config/examples/cors.yml

```diff
@@ -1,13 +1,13 @@
 cors: &default
   enabled: false
   allow:
-  - origins: '*'
-    resources:
-    - !ruby/regexp /\.(?:woff2?|otf|ttf|svg|eot)$/
-    headers: 'x-requested-with'
-    methods: :get
-    max_age: 3628800
-    credentials: false
+    - origins: '*'
+      resources:
+        - !ruby/regexp /\.(?:woff2?|otf|ttf|svg|eot)$/
+      headers: 'x-requested-with'
+      methods: :get
+      max_age: 3628800
+      credentials: false
```

Comment:
why change indentation here?

Reply:
Just a personal preference to add indentation for arrays :grimacing: 
I can revert though

Reply:
yes, please lets limit just cosmetic changes, it's a long PR anyway. btw my personal preference is to keep indentation as little as possible